### ISSUE TITLE
1.9.0-rc1 : network bridge not found error when cluster-store couldnt be reached during bootup

### ISSUE THREAD

**Issue by mavenugo** [docker/libnetwork#651]
When the cluster-store is configured, but is not reachable during the daemon bootup (this is very common when the store is a container)

time="2015-10-13T20:27:13.871902450Z" level=debug msg="Starting container 8b863644e34d40540897b0fe1e098c0252e9661e83ae4b634c9952292ecd6a52" 
time="2015-10-13T20:27:13.887254679Z" level=error msg="failed to get networks for scope global: client: etcd cluster is unavailable or misconfigured" 
time="2015-10-13T20:27:13.887358363Z" level=error msg="failed to get networks for scope global: client: etcd cluster is unavailable or misconfigured" 
time="2015-10-13T20:27:13.887378169Z" level=warning msg="Could not connect container aecd8f468ed2b631113d40017e406f691e220183a018f9373e4b661a0cd14633 : network bridge not found"

**Comment by mavenugo** [comment]
hitting another variant of the same issue during a reboot -

vagrant@net-2:/$ sudo /usr/bin/docker daemon --cluster-store=consul://192.168.33.10:8500 --cluster-advertise=192.168.33.12:0 &
[1] 1784
vagrant@net-2:/$ INFO[0000] API listen on /var/run/docker.sock
INFO[0000] [graphdriver] using prior storage driver "aufs"
INFO[0000] Initializing discovery without TLS
ERRO[0000] discovery error: Unexpected watch error
INFO[0000] Firewalld running: false
ERRO[0000] Registering as "192.168.33.12:0" in discovery failed: cannot set or renew session for ttl, unable to operate on sessions
ERRO[0000] failed to get networks for scope global: Get http://192.168.33.10:8500/v1/kv/docker/network/v1.0/network/?consistent=: dial tcp 192.168.33.10:8500: connection refused
FATA[0000] Error starting daemon: Error initializing network controller: Error creating default "bridge" network: failed to allocate gateway (172.17.0.1): No available addresses on this pool

**Comment by hd-deman** [comment]
same for me. can't start docker because I start consul after that.

**Comment by jameBoy** [comment]
are you solve your problem ? if not, you can see [#18113](https://github.com/docker/docker/issues/18113). And I also get the problem.I delete the /var/lib/docker/network/files/local-kv.db. and restart the deamon.

**Comment by mavenugo** [comment]
This issue was resolved during the 1.9.0 release.

**Comment by auzias** [comment]
Hi, I'm running docker 1.9.1 with:
`/usr/bin/docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock --cluster-advertise eth0:2375 --cluster-store consul://127.0.0.1:8500`, note that `consul` is not running, and have this error in the log:
````
Jan 13 09:50:04 bugs20 docker[29356]: time="2016-01-13T09:50:04.203417359+01:00" level=error msg="Registering as \"192.168.196.20:2375\" in discovery failed: cannot set or renew session for ttl, unable to operate on sessions"
Jan 13 09:50:19 bugs20 docker[29356]: time="2016-01-13T09:50:19.056980231+01:00" level=error msg="discovery error: Unexpected watch error"
````

I'm unsure that this is related with this exact issue but I didn't want to create a new one for now. Similar ones are https://github.com/docker/docker/issues/18009 and https://github.com/docker/libnetwork/issues/651.

I also found a similar error on the [docker forum](https://forums.docker.com/t/startup-error-using-clustering-in-aws/5437).
